backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3333,4 +3333,102 @@ databaseChangeLog:
                   constraints:
                     nullable: true
                     foreignKeyName: fk__facility__default_specimen_type
-                    references: specimen_type(internal_id)
\ No newline at end of file
+                    references: specimen_type(internal_id)
+  - changeSet:
+      id: add-disease-table
```

Comment:
I think we have been matching the ids and tags `add-supported-disease-table`

Reply:
Good catch, updated!